pennylane/capture/meta_type.py

```diff
@@ -0,0 +1,129 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
I _think_ this is the convention of what we put in copyright statements for new files?
```suggestion
# Copyright 2024 Xanadu Quantum Technologies Inc.
```